src/main/scala/it/pagopa/interop/attributeregistryprocess/common/readmodel/TotalCountResult.scala

```diff
@@ -0,0 +1,11 @@
+package it.pagopa.interop.attributeregistryprocess.common.readmodel
```

Comment:
[same](https://github.com/pagopa/interop-be-attribute-registry-process/pull/1/files#r1135226074)